RELEASE_NOTES.md

```diff
@@ -8,6 +8,42 @@ This document contains release notes for the current major release and all patch
 For prior releases, see
 [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
+## 12.12.0 (2017-01-26)
+
+This release addresses a number of bugs, the most notable are describe
+below.
+
+- `chef-server-ctl backup` correctly backs up configuration
+
+Starting in version 12.10.0, a bug in the backup command produced
+backups that did not include the configuration data in the resulting
+tarball. This bug is now resolved. We recommend taking a new backup
+after upgrading to 12.12.0.
+
+- Search respects rows parameter when using ElasticSearch
+
+When configured to use ElasticSearch, the Chef Server now correctly
+respects the `rows` parameter to search requests rather than returning
+all rows.
+
+We recommend upgrading to 12.12.0 for all users of Chef Backend, which
```

Comment:
s/, which uses.*/to receive this search fix.

(Second time adding this. The version in the review itself is missing...)